src/main/java/gov/usgs/wma/waterdata/PreProcess.java

```diff
@@ -28,26 +27,27 @@ public ResultObject apply(RequestObject request) {
 
 	@Transactional
 	protected ResultObject processRequest(RequestObject request) {
-		LOG.debug("json_data_id: {}", request.getId());
+		Long jsonDataId = request.getId();
```

Comment:
I'm torn between getting the jsonDataId & partitionNumber here vs just passing the request object to each method. With just the two parameters it's ok, but if more would be added the method signatures would continue to expand...

Reply:
Yeah I was juggling this as well... I think from a visual perspective, the easier these just-after-the-handler classes are to look at, the better.  Maybe splitting out the request properties in the shared DAO method that actually launches the jdbc call would be the best place to split it out.

Reply:
maybe just put the property's get method in the debug and eliminate the two variable here too?